test/functional/c/test_symmetry.cpp

```diff
@@ -21,21 +21,23 @@ TEST(Symmetry, test_get_lattice_symmetry_layer) {
     };
     double positions[1][3] = {{0, 0, 0}};
     const int types[1] = {0};
-    const int aperiodic_axis = 2;
     const double symprec = 1e-5;
     const double angle_tolerance = -1;
 
     cell = cel_alloc_cell(size, NOSPIN);
     ASSERT_NE(cell, nullptr);
     ASSERT_EQ(spg_get_error_code(), SPGLIB_SUCCESS);
-    cel_set_layer_cell(cell, lattice, positions, types, aperiodic_axis);
+    for (int aperiodic_axis = 0; aperiodic_axis < 3; aperiodic_axis++) {
+        cel_set_layer_cell(cell, lattice, positions, types, aperiodic_axis);
 
-    // Bravais group of the two-dimensional lattice is 4/mmm
-    symmetry = sym_get_operation(cell, symprec, angle_tolerance);
-    ASSERT_EQ(symmetry->size, 16);
+        // Bravais group of the two-dimensional lattice is 4/mmm
+        symmetry = sym_get_operation(cell, symprec, angle_tolerance);
+        ASSERT_EQ(symmetry->size, 16);
+
+        sym_free_symmetry(symmetry);
+        symmetry = NULL;
+    }
 
-    sym_free_symmetry(symmetry);
     cel_free_cell(cell);
-    symmetry = NULL;
     cell = NULL;
```

Comment:
```suggestion
    cell = nullptr;
```

Reply:
I am not very familiar with C++. Should I change `NULL` at line 13 and 14 to `nullptr`?

Reply:
Yes, wherever you find it. In most cases you will not find differences with C/C++ interface, but if we get around to migrating the code to C++ it will become relevant.

Reply:
I see, Thanks! I have modified the test and rebased the branch. Please note that a force push appeared.